### PR TITLE
s/master/control-plane in 

### PR DESCRIPTION
**Pull request by kwmonroe**
Our CI often checks config, status, etc of certain units.  Now that #874 has made the switch to deploy artifacts from charmhub, we need to update the `-master` app name to `-control-plane`.

I've set the branches for validate jobs to use this PR's branch as the source of truth. This looks to be [working well](https://jenkins.canonical.com/k8s/job/validate-ck/804/arch=amd64,channel=edge,node=runner-validate,series=focal,snap_version=1.23%2Fedge/console):
```
23:12:30 [validate-ck-amd64-focal-1.23-edge] ========= 22 passed, 13 skipped, 11168 warnings in 6714.91s (1:51:54) ==========
```

Reviewers: please approve but stop short of merging. Once approved, I'll re-`jjb` the jobs to switch back to the `main` branch before merging.

### CODE REVIEW

File: jobs/integration/test_lxd_profile_deploy.py [hunk 1]
```diff
@@ -68,13 +68,17 @@ def trim_profile(profile):
...
+)
 async def test_lxd_profiles(model, charm_name):
     app = model.applications[charm_name]
     await check_charm_profile_deployed(app, charm_name)
 
 
-@pytest.mark.parametrize("charm_name", ("kubernetes-worker", "kubernetes-master"))
+@pytest.mark.parametrize(
+    "charm_name", ("kubernetes-worker", "kubernetes-control-plane")
+)
```
Comment (addyess) [inline jobs/integration/test_lxd_profile_deploy.py:81]: thanks `black`

File: jobs/integration/validation.py [hunk 2]
```diff
@@ -252,7 +252,7 @@ async def test_auth_file_propagation(model, tools):
 async def test_status_messages(model):
     """Validate that the status messages are correct."""
     expected_messages = {
-        "kubernetes-master": "Kubernetes master running.",
+        "kubernetes-control-plane": "Kubernetes control-plane running.",
```
Comment (addyess) [inline jobs/integration/validation.py:255]: Good, glad you've caught this one. I believe there are a few integration tests sprinkled through `charmed-kubernetes` org that need this adjustment

File: jobs/integration/validation.py [hunk 3]
```diff
@@ -964,9 +964,9 @@ async def test_kubelet_extra_config(model, tools):
 
     # wait for and validate new maxPods value
     click.echo("waiting for nodes to show new pod capacity")
-    master_unit = model.applications["kubernetes-master"].units[0]
+    master_unit = model.applications["kubernetes-control-plane"].units[0]
     while True:
-        cmd = "/snap/bin/kubectl --kubeconfig /root/.kube/config -o yaml get node"
+        cmd = "/snap/bin/kubectl --kubeconfig /root/.kube/config -o yaml get node -l 'juju-application=kubernetes-worker'"
```
Comment (addyess) [inline jobs/integration/validation.py:969]: is this so you ignore the control-plane?

**Review by addyess — APPROVED** [review]
Yep -- these are all things i knew were coming and its nice to see them here

**Comment by addyess** [comment]
would you consider adding this to the `.github/workflow/*.yaml`

```
  Inclusive-naming-check:
    runs-on: ubuntu-latest
    steps:
      - name: Check out code
        uses: actions/checkout@v2
      - name: woke
        uses: canonical-web-and-design/inclusive-naming@main
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          reporter: github-pr-check
          fail-on-error: true
```

File: jobs/integration/validation.py (hunk 3 above)
Comment (kwmonroe) [reply jobs/integration/validation.py:969]: Correct; the test sets config on `k8s-worker` explicitly, so the verify section should only be looking at those nodes.

**Pull request merged**